client/stylesheets/_responsive.scss

```diff
@@ -221,13 +257,23 @@
         margin: 8px 0 10px 0 !important;
         font-size: 1rem !important;
     }
+
     .searchResultsCardSize, .searchResultsCardCondition, .searchResultsCardLocation {
         margin: 12px 0 !important;
         font-size: 1rem !important;
     } 
+
     .searchResultsCardName {
         font-size: 0.95rem !important;
     }
+
+    .profileUserRating ul li span {
+        font-size: 0.8rem !important;
+    }
+
+    .profileUserImage img {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `profileUserImage` should be written in lowercase with hyphens